cluster/src/fpindexcluster/proxy_service.py

```
@@ -0,0 +1,267 @@
+        # Index deletion would require purging all messages for the index
+        # This is a more complex operation - for now just return success
+        return web.json_response({"status": "deleted", "index": index_name})
+    
+    # Single fingerprint operations
+    def _validate_fingerprint_id(self, fp_id_str: str) -> tuple[bool, int]:
+        """Validate that fingerprint ID is a valid 32-bit unsigned integer"""
+        try:
+            fp_id = int(fp_id_str)
+            if 0 <= fp_id <= 0xFFFFFFFF:  # 32-bit unsigned integer range
+                return True, fp_id
+            return False, 0
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)